Gopkg.toml

```diff
@@ -13,11 +14,6 @@ required = [
   revision = "0ca9ea5df5451ffdf184b4428c902747c2c11cd7"
   name = "github.com/ghodss/yaml"
 
-[[constraint]]
-  # Also defined in WORKSPACE
-  revision = "f8815663de1e64d57cdd4ee9e2b2fa96977a030e"
-  name = "github.com/go-resty/resty"
-
```

Comment:
Where did this change come from?

Reply:
me doing `dep ensure -add github.com/golang/protobuf/descriptor` for some reason removed it and I can't recreate why at this point, reverted that change

Reply:
`dep` works in mysterious ways